app/controllers/users/nbp_wallet_controller.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module Users
+  class NbpWalletController < ApplicationController
+    CONNECTOR_URL = 'http://localhost:80'
+    API_KEY = 'myapikey'
+    DISPLAY_NAME = 'CodeHarbor'
+    REQUIRED_ATTRIBUTES = %w[GivenName Surname EMailAddress].freeze # https://enmeshed.eu/integrate/attribute-values
```

Comment:
This one would be part of the "mapping" I mentioned in the `NbpWalletController`. If we create such a mapping, these strings would be the "values" of the mapping.

Reply:
What kind of mapping do you mean?

Reply:
Can we make it similar to OmniAuth?

```rb
      def info
        {
          email: @attributes['EMailAddress'],
          first_name: @attributes['GivenName'],
          last_name: @attributes['Surname'],
        }
      end
```

My goal would be to have this single place to change if they change something, e.g., rename `Surname` to `FamilyName`. And extracting in a dedicated method seems good to achieve that goal, does it?

Reply:
I included such a mapping in the most recent version of the commit.